controllers/auth.controller.js

```diff
@@ -3,8 +3,8 @@ const tokenService = require("../service/token.service");
 
 const signUp = async (req, res, next) => {
     try {
-        const { name, email, password } = req.body;
-        const data = await userService.signUp(name, email, password);
+        const { first_name, last_name, phone_number, email, password } = req.body;
+        const data = await userService.signUp(first_name, last_name, phone_number, email, password);
```

Comment:
@ikadushkina your signUp function takes 4 arguments, it's easy to confuse the order and make a mistake. For the future I recommend to take a look at the [RORO pattern](https://www.freecodecamp.org/news/elegant-patterns-in-modern-javascript-roro-be01e7669cbd/). It will make you functions that takes a lot of parameters easier to be called, like `userService.signUp({ first_name, last_name, phone_number })`. And pretty good it will look with lodash, like `userService.signUp(_.pick(req.body, ['first_name', 'last_name', 'phone_number']))`